packages/inter-protocol/src/vaultFactory/liquidation.js

```diff
@@ -2,143 +2,151 @@
 
 import { E } from '@endo/eventual-send';
 import { AmountMath } from '@agoric/ertp';
-import {
-  ceilMultiplyBy,
-  makeRatio,
-  offerTo,
-} from '@agoric/zoe/src/contractSupport/index.js';
 import { makeTracer } from '@agoric/internal';
+import { TimeMath } from '@agoric/time';
+import { atomicRearrange } from '@agoric/zoe/src/contractSupport/index.js';
+import { makeScalarMapStore } from '@agoric/store';
+
+import { AUCTION_START_DELAY, PRICE_LOCK_PERIOD } from '../auction/params.js';
 
 const trace = makeTracer('LIQ', false);
 
-/**
- * Threshold to alert when the price level falls enough that the vault
- * with the highest debt to collateral ratio will no longer be valued at the
- * liquidationMargin above its debt.
- *
- * @param {Ratio} highestDebtRatio
- * @param {Ratio} liquidationMargin
- */
-const liquidationThreshold = (highestDebtRatio, liquidationMargin) =>
-  ceilMultiplyBy(
-    highestDebtRatio.numerator, // debt
-    liquidationMargin,
-  );
+/** @typedef {import('@agoric/time/src/types').TimerService} TimerService */
+/** @typedef {import('@agoric/time/src/types').TimerWaker} TimerWaker */
+/** @typedef {import('@agoric/time/src/types').CancelToken} CancelToken */
+/** @typedef {import('@agoric/time/src/types').RelativeTimeRecord} RelativeTimeRecord */
 
 /**
- * @param {ERef<PriceAuthority>} priceAuthority
- * @param {Ratio} highestDebtRatio
- * @param {Ratio} liquidationMargin
+ * @param {ERef<import('../auction/auctioneer.js').AuctioneerPublicFacet>} auctionPublicFacet
+ * @param {ERef<TimerService>} timer
+ * @param {TimerWaker} priceLockWaker
+ * @param {TimerWaker} liquidationWaker
+ * @param {TimerWaker} reschedulerWaker
  */
-const makeQuote = (priceAuthority, highestDebtRatio, liquidationMargin) => {
-  return E(priceAuthority).mutableQuoteWhenLT(
-    highestDebtRatio.denominator, // collateral
-    liquidationThreshold(highestDebtRatio, liquidationMargin),
+const scheduleLiquidationWakeups = async (
+  auctionPublicFacet,
+  timer,
+  priceLockWaker,
+  liquidationWaker,
+  reschedulerWaker,
+) => {
+  const [schedules, params] = await Promise.all([
+    E(auctionPublicFacet).getSchedules(),
+    E(auctionPublicFacet).getGovernedParams(),
+  ]);
+  const { startTime } = schedules.nextAuctionSchedule;
+
+  trace('SCHEDULE', schedules.nextAuctionSchedule);
+
+  /** @type {RelativeTimeRecord} */
+  // @ts-expect-error Casting
+  // eslint-disable-next-line no-restricted-syntax
```

Comment:
seems like Jessie is wrong here. It thinks it's a computed property but it's a constant.
```suggestion
  // eslint-disable-next-line no-restricted-syntax -- https://github.com/Agoric/eslint-config-jessie/issues/33
```

Reply:
added